### PR TITLE
Italian contest loader is able to load contests with no participants

### LINKED ISSUE #1192: Italian contest loader fails with no participants
**Description:**

The italian contest loader fails when the contest has no participants.

**Steps to reproduce:**
1. Download the contest example (https://github.com/cms-dev/con_test)
2. Open `contest.yaml` and remove `users` section
3. Import the contest (using `cmsImportContest -L italy_yaml`)

**Expected:**

The contest is added to CMS with all the problems but no participants.

**Actual:**

The application crashes

### System Information

**CMS version:** tested in `v1.4` (but the bug remains in `master`)
**Was CMS installed:** yes
**Using a virtualenv:** no

### PR DESCRIPTION
**Pull request by MarcoUCM**
Fixes #1192 (Italian contest loader fails with no participants)

Please, consider applying the patch to v1.4

### CODE REVIEW

File: cmscontrib/loaders/italy_yaml.py [hunk 1]
```diff
@@ -219,6 +219,7 @@ def get_contest(self):
 
         tasks = load(conf, None, ["tasks", "problemi"])
         participations = load(conf, None, ["users", "utenti"])
+        participations = [] if participations is None else participations;
```
Comment (andreyv) [inline cmscontrib/loaders/italy_yaml.py:222]:
```suggestion
        participations = [] if participations is None else participations
```
Semicolon is not needed here.

**Review by andreyv — CHANGES_REQUESTED** [review]
Thanks, the fix looks appropriate.

Please see one minor comment below — I don't have access to edit this branch.

File: cmscontrib/loaders/italy_yaml.py (hunk 1 above)
Comment (MarcoUCM) [reply cmscontrib/loaders/italy_yaml.py]: Ups. Done.

**Review by andreyv — APPROVED** [review]
Thanks!

**Pull request merged**